### PR TITLE
CAP-73 #comment add supports to list route for column selecting

### PR DESCRIPTION
**Pull request by karanokara**

### CODE REVIEW

**Comment by anhaley** [comment]
When I clone this and do `curl localhost:800/list?table=intake&column=dba`, it creates a background job instead of displaying the results immediately. Any idea why this is happening? Is this expected behavior?
image
It also doesn't seem to be listing single columns, though I saw it work during your demo. Is my syntax wrong?

File: server.py [hunk 1]
```diff
@@ -39,7 +39,17 @@ def dump_table():
         return make_response(jsonify('Table name not supplied.'), 400)
     try:
         # TODO: once authentication is in place, restrict the tables that can be listed here
-        table_info_obj = driver.get_table(table_name)
+        columns = request.args.get('column', '')
+        if columns == '':
+            columns = None
+        else:
+            app.logger.info('columns')
```
Comment (anhaley) [inline server.py:46]: Is there a reason to keep these logging statements?

**Comment by smitchell1980** [comment]
> When I clone this and do `curl localhost:800/list?table=intake&column=dba`, it creates a background job instead of displaying the results immediately. Any idea why this is happening? Is this expected behavior?
> image
> It also doesn't seem to be listing single columns, though I saw it work during your demo. Is my syntax wrong?

I was getting the same thing.  Its due to the single '&'   tried double && but still getting a full list of all the entries from the table not jsut the column

File: server.py (hunk 1 above)
Comment (karanokara) [reply server.py:46]: Not my code, I don't know who wrote it.

**Comment by karanokara** [comment]
curl treats it as 2 arguments, need to wrap it with "",
curl "localhost:800/list?table=intake&column=dba+MRL"

File: server.py (hunk 1 above)
Comment (karanokara) [reply server.py:47]: Oh, that is for logging the arguments, keep there for debugging, it doesn't matter.

**Pull request merged**